java/demo/src/com/google/phonenumbers/PhoneNumberParserServlet.java

```diff
@@ -212,6 +273,9 @@ private StringBuilder getOutputForSingleNumber(
     output.append("Language entered: " + StringEscapeUtils.escapeHtml(languageCode) +
         (regionCode.isEmpty() ? "" : " (" + StringEscapeUtils.escapeHtml(regionCode) + ")")
         + "<BR>");
+    output.append("[<a href=\""
+        + getPermaLinkURL(phoneNumber, defaultCountry, languageCode, regionCode)
+        + "\">link to this result page</a>]<br>");
```

Comment:
@hagbard - Any opinion on this link? @roes thinks we should remove it since it'll already be in the URL.